res/contribute.json

```diff
@@ -9,15 +9,26 @@
       "participate": {
         "home": "https://github.com/firefox-devtools/profiler/blob/master/CONTRIBUTING.md",
         "docs": "https://github.com/firefox-devtools/profiler/tree/master/docs-developer",
-        "irc": "irc://irc.mozilla.org/#devtools",
+        "matrix": "https://chat.mozilla.org/#/room/#profiler:mozilla.org",
         "slack": "https://devtools-html-slack.herokuapp.com/",
+        "irc": "irc://irc.mozilla.org/#devtools",
         "irc-contacts": [
           "gregtatum",
           "julienw",
           "florian",
           "canova",
           "gerald"
-        ]
+        ],
+        "chat": {
+          "url": "https://chat.mozilla.org/#/room/#profiler:mozilla.org",
+          "contacts": [
+            "@gregtatum:mozilla.com",
+            "@julienw:mozilla.com",
+            "@florian:mozilla.com",
+            "@canova:mozilla.com",
+            "@gerald:mozilla.com"
+          ]
+        }
```

Comment:
For this I followed the discussion in https://github.com/mozilla/contribute.json/issues/106.
I also kept the outdated IRC links because Mozilla Observatory hasn't been updated to support this new property yet.